src/audio/module_adapter/module_adapter.c

```diff
@@ -963,7 +963,27 @@ int module_get_large_config(struct comp_dev *dev, uint32_t param_id, bool first_
 						  (uint8_t *)data, fragment_size);
 	return 0;
 }
+
+int module_adapter_get_attribute(struct comp_dev *dev, uint32_t type, void *value)
```

Comment:
If we are passing a void * we need to also pass size, i.e. to make sure caller point is big enough for memcpy_s()